docs/ci/command.md

```diff
@@ -12,7 +12,7 @@
 
 | ci pipeline                                 | Commands                                                    | Only triggered by command |
 | ------------------------------------------- | ----------------------------------------------------------- | ------------------------- |
-| idc-jenkins-ci-ticdc/unit-test              | /run-unit-test <br />/run-all-tests<br />/merge             | No                        |
+| jenkins-ticdc/verify                        | /run-verify-ci<br />/run-all-tests<br />/merge              | No                        |
```

Comment:
I understand that this task is still needed on other branches?

Reply:
nice catch, i will add `unit-test` back in next commit

Reply:
@hi-rustin added in https://github.com/pingcap/ticdc/pull/3586/commits/202e5c09371cf84c8dabe3de76aa3a1937231559 do you have more suggestions? if not, i will merge this pr